### PR TITLE
Add User Field and Logic for Pause Pear (2/2)

### PR DESCRIPTION
**Pull request by jzs27**
## Overview

Added a celery beat that checks if the user's `pause_expiration` date has passed.

## Changes Made

- Added `shared_task` in /person/tasks.py  that sets the `is_paused` = False and `pause_expiration` = None if `pause_expiration` date has passed.

## Test Coverage

Manual Testing in Postman

## Related PRs or Issues (delete if not applicable)

#96

### CODE REVIEW

File: src/match/tasks.py [hunk 1]
```diff
@@ -1,75 +1,60 @@
...
-    task="match.tasks.matcher",
-)
+# # Get Pear algorithm
+# current_dir = os.path.dirname(os.path.abspath(__file__))
+# submodule_path = f"{current_dir}/../../pear-algorithm/src"
+# sys.path.insert(0, submodule_path)
+# from main import main
+
+# app = Celery()
+
```
Comment (alanna-zhou) [inline src/match/tasks.py:8]: why is this file all commented out?

**Review by alanna-zhou — APPROVED** [review]
nice!

**Review by chalo2000 — CHANGES_REQUESTED** [review]
Good job Jessica, some comments.
1) I am equally confused with Alanna with why an entire file is commented out, if this was for testing purposes make sure you tested properly, talking about what you did in the Testing Coverage section of the PR, and uncomment (or delete if unused) before pushing
2) We need to modify the length of the cronjob, if you got it working with the interval that's great but we should be running this cronjob at certain points in the day - this we can also test similar to how you did with the interval schedule since the nice thing about crontab is you can do intervals such as [a 5 minute interval](https://crontab.guru/#*/5_*_*_*_*) and also scheduled times like [at midnight and noon](https://crontab.guru/#0_0,12_*_*_*).

File: src/match/tasks.py (hunk 1 above)
Comment (noah-solomon) [reply src/match/tasks.py:8]: this is my fault, this is gonzalo's previous implementation of matching which I improved on a different branch & PR [(noah/auto-matcher)](https://github.com/cuappdev/pear-django-backend/pull/97). at my meeting tomorrow with @chalo2000 and @jzs27 we're going to merge in that branch, deploy to dev, and see if it works

tl;dr: this file was messing up our other celery stuff so I should've just deleted it alltogether

**Comment by noah-solomon** [comment]
> Good job Jessica, some comments.
> 
> 1. I am equally confused with Alanna with why an entire file is commented out, if this was for testing purposes make sure you tested properly, talking about what you did in the Testing Coverage section of the PR, and uncomment (or delete if unused) before pushing
> 2. We need to modify the length of the cronjob, if you got it working with the interval that's great but we should be running this cronjob at certain points in the day - this we can also test similar to how you did with the interval schedule since the nice thing about crontab is you can do intervals such as [a 5 minute interval](https://crontab.guru/#*/5_*_*_*_*) and also scheduled times like [at midnight and noon](https://crontab.guru/#0_0,12_*_*_*).

regarding 1 - peep my reply to alanna above, this will be addressed in my other PR

File: src/match/tasks.py (hunk 1 above)
Comment (alanna-zhou) [reply src/match/tasks.py:8]: sg
